Changes.md

```diff
@@ -0,0 +1,20 @@
+# Changes
+
+## October 24, 2022
+
+* Automatic common/ update #155 - this new update include the
+  addition of global.clusterVersion as a new helm variable which represents the OCP
+  Major.Minor cluster version. By default now a user can add a
+  values-<ocpversion>-<clustergroup>.yaml file to have specific cluster version
+  overrides (e.g. values-4.10-hub.yaml). Will need Validated Patterns Operator >= 0.0.6
+  when deploying with the operator. Note: When using the ArgoCD Hub and spoke model,
+  you cannot have spokes with a different version of OCP than the hub.
+
```

Comment:
I'd drop all this and just add a big fat line above that ACM 2.6 is now required

Reply:
Ack